components/footer/src/atoms/copyright/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+import Link from '@govuk-react/link';
+import { FOOTER_LINK, FOOTER_LINK_HOVER } from 'govuk-colours';
+
+const crestImageWidth2x = 250;
+const crestImageHeight2x = 204;
+const crestImageWidth = crestImageWidth2x / 2;
+const crestImageHeight = crestImageHeight2x / 2;
+
+const StyledCopyright = styled(Link)({
+  display: 'inline-block',
+  minWidth: crestImageWidth,
+  paddingTop: crestImageHeight + 10,
+  backgroundImage: "url('https://design-system.service.gov.uk/assets/images/govuk-crest.png')", // Needs changing
```

Comment:
this url should be optionally provided as a prop and default to no image

e.g. when we publish our storybook we shouldn't use the crest for the default stories but perhaps show how it could be configured to use a deployed crest image